session1/3-challenge/5_string.rb

```diff
@@ -1,8 +1,9 @@
 # Given a string, replace every instance of sad to happy
-# 
+#
 # add_more_ruby("The clowns were sad.")         # => "The clowns were happy."
 # add_more_ruby("The sad dad said sad stuff.")  # => "The happy dad said happy stuff."
 # add_more_ruby("Sad times are ahead!")         # => "Happy times are ahead!"
 
 def add_more_ruby(string)
-end
\ No newline at end of file
+  string.downcase.gsub('sad', 'happy').capitalize
```

Comment:
Hi,
Interesting solution, actually. It would, unfortunatly, not work if you have a "Sad" mid sentence, since capitalize only upcases the first char of a string. So if you have something like "He said:'Sad!'", you will get "He said: 'happy!'"